handbook/business-operations/README.md

```diff
@@ -858,6 +858,35 @@ To make a request of the business operations department, [create an issue using
 
 > If you're not sure that your request can wait that long, then please ask for urgent help in our group Slack channel: `#g-business-operations`.  Only use this approach or at-mention contributors in business operations directly in urgent situations.  Otherwise, create an issue.
 
+## Contract review
+
+**If a document is ready for signature and does not need to be reviewed or negotiated, you can skip the review process and use the signature process documented further down in this handbook section.**
+
+**To get a contract reviewed:**
+
+Upload the agreement to the [drafts folder](https://drive.google.com/drive/folders/1G1JTpFxhKZZzmn2L2RppohCX5Bv_CQ9c) in GoogleDrive.
+
+Complete the contract review issue template in GitHub, being sure to include the link to the document you uploaded and using the Calendly link in the issue template to schedule time to discuss the agreement with Nathan Holliday.
```

Comment:
light up text with a link to https://fleetdm.com/handbook/business-operations#intake